### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by nirinchev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,9 @@
-x.y.z (TBD)
+1.1.0 (2017-03-02)
 ------------------
 
-### Breaking Changes
-
 ### Enhancements
 - Added Azure Active Directory (AzureAD) credentials provider. (#1254)
```
Comment (AndyDentFree) [inline CHANGELOG.md]: are there weaver fixes to also mention in the changelog?

File: Tests/ThreeLayerRealmXFNuGet/PurePCLViewModel/packages.config [hunk 2]
```diff
@@ -2,8 +2,8 @@
 <packages>
   <package id="DotNetCross.Memory.Unsafe" version="0.2.2" targetFramework="portable-net45+win+wpa81+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarintvos10+xamarinwatchos10+xamarinios10" />
   <package id="Fody" version="1.29.4" targetFramework="portable-net45+win+wpa81+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" developmentDependency="true" />
-  <package id="Realm" version="1.0.4" targetFramework="portable45-net45+win8+wp8+wpa81" developmentDependency="true" />
-  <package id="Realm.Database" version="1.0.4" targetFramework="portable45-net45+win8+wp8+wpa81" />
+  <package id="Realm" version="1.1.0" targetFramework="portable45-net45+win8+wp8+wpa81" developmentDependency="true" />
```
Comment (AndyDentFree) [inline Tests/ThreeLayerRealmXFNuGet/PurePCLViewModel/packages.config]: remove developmentDependency="true" from Realm

File: Tests/ThreeLayerRealmXFNuGet/ThreeLayerRealmXFNuGet/ThreeLayerRealmXFNuGet.Droid/packages.config [hunk 3]
```diff
@@ -2,8 +2,8 @@
 <packages>
   <package id="DotNetCross.Memory.Unsafe" version="0.2.2" targetFramework="MonoAndroid60" />
   <package id="Fody" version="1.29.4" targetFramework="MonoAndroid60" developmentDependency="true" />
-  <package id="Realm" version="1.0.4" targetFramework="monoandroid70" developmentDependency="true" />
-  <package id="Realm.Database" version="1.0.4" targetFramework="monoandroid70" />
+  <package id="Realm" version="1.1.0" targetFramework="monoandroid71" developmentDependency="true" />
```
Comment (AndyDentFree) [inline Tests/ThreeLayerRealmXFNuGet/ThreeLayerRealmXFNuGet/ThreeLayerRealmXFNuGet.Droid/packages.config]: remove developmentDependency="true" from Realm

File: examples/Benchmarkr/Droid/packages.config [hunk 4]
```diff
@@ -9,7 +9,8 @@
   <package id="Microsoft.Win32.Primitives" version="4.3.0" targetFramework="monoandroid70" />
   <package id="NETStandard.Library" version="1.6.1" targetFramework="monoandroid70" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="monoandroid70" />
-  <package id="Realm.Database" version="1.0.4" targetFramework="monoandroid70" developmentDependency="true" />
+  <package id="Realm" version="1.1.0" targetFramework="monoandroid71" />
+  <package id="Realm.Database" version="1.1.0" targetFramework="monoandroid71" developmentDependency="true" />
```
Comment (AndyDentFree) [inline examples/Benchmarkr/Droid/packages.config]: remove developmentDependency="true" from Realm.Database

File: examples/Benchmarkr/iOS/packages.config [hunk 5]
```diff
@@ -10,7 +10,8 @@
   <package id="Microsoft.Win32.Primitives" version="4.3.0" targetFramework="xamarinios10" />
   <package id="NETStandard.Library" version="1.6.1" targetFramework="xamarinios10" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="xamarinios10" />
-  <package id="Realm.Database" version="1.0.4" targetFramework="xamarinios10" developmentDependency="true" />
+  <package id="Realm" version="1.1.0" targetFramework="xamarinios10" />
+  <package id="Realm.Database" version="1.1.0" targetFramework="xamarinios10" developmentDependency="true" />
```
Comment (AndyDentFree) [inline examples/Benchmarkr/iOS/packages.config]: remove developmentDependency="true" from Realm.Database

File: CHANGELOG.md (hunk 1 above)
Comment (nirinchev) [inline CHANGELOG.md]: The weaver is something we haven't released, so the fixes are not visible. But I'll add a mention that we're using a new build system.

**Pull request merged**